package.json

```diff
@@ -13,11 +13,14 @@
     "testcafe": "^1.8.8"
   },
   "scripts": {
-    "examples": "testcafe chrome,firefox examples -a \"node server/index.js\"",
+    "examples": "testcafe chrome examples -a \"node server/index.js\"",
     "mock-camera-microphone-access": " testcafe --hostname localhost \"chrome --use-fake-ui-for-media-stream --use-fake-device-for-media-stream\" detached-examples/mock-camera-microphone-access/mock-camera-microphone-access.js",
-    "test": "npm run examples && npm run mock-camera-microphone-access"
+    "test": "npm run lint & npm run examples && npm run mock-camera-microphone-access",
+    "lint": "eslint \"*(detached-)examples/**/*.js\""
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
Remove the `babel-eslint` module.

Reply:
are you sure? Eslint is configured to use it.

**.eslintrc**:

`"parser": "babel-eslint",`

Reply:
Good catch.

Remove the `"parser": "babel-eslint"` directive from `.eslintrc` file to use the default parser.

Reply:
Done. For eslint to recognize arrow functions and import statements, I had to tweak `eslintrc`. See [this commit](https://github.com/DevExpress/testcafe-examples/pull/53/commits/cc13cc1ababc0690de532aa9489af1b9fe991985#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822)